src/functionalTest/resources/responses/response-cic-submit-case-notification-failed.json

```diff
@@ -0,0 +1,3 @@
+{
+  "confirmation_header": "# Application Received notification failed %n## Please resend the notification"
```

Comment:
this can be replaced with the shortened version using
```
assertThatJson(response.asString())
            .inPath("$.confirmation_header")
            .isEqualTo("# Application Received notification failed %n## Please resend the notification");
```